lib/integration/framework/Test/Integration/Framework/TestData.hs

```diff
@@ -335,15 +334,6 @@ errMsg403InvalidConstructTx =
      \any payments, withdrawals, delegations, metadata nor minting. \
      \Include at least one of them."
 
-errMsg403MinUTxOValue :: String
-errMsg403MinUTxOValue = unwords
-    [ "One of the outputs you've specified has an ada quantity that is"
-    , "below the minimum required. Either increase the ada quantity to"
-    , "at least the minimum, or specify an ada quantity of zero, in"
-    , "which case the wallet will automatically assign the correct"
-    , "minimum ada quantity to the output."
-    ]
```

Comment:
This is just a duplication of:

https://github.com/cardano-foundation/cardano-wallet/blob/092f2f887bda39998fdd145f4a9206d4e853f0c3/lib/wallet/api/http/Cardano/Wallet/Api/Http/Server/Error.hs#L1033-L1039